sw/device/silicon_creator/mask_rom/docs/_index.md

```diff
@@ -1,7 +1,5 @@
 ---
 title: "Mask ROM"
-aliases:
-- /sw/device/silicon_creator/mask_rom/boot
```

Comment:
I **think** this change and the name change from `index.md` to `_index.md` are required to get everything to work nicely but I haven't double checked yet.